programmers/k-th-number.js

```diff
@@ -0,0 +1,15 @@
+function solution(arry, commands){
+    const cd_len = commands.length;
```

Comment:
캐멀케이스로 지어주세요 
하지만 cdLen도 좋은 변수명은 아닌거같아요.
```suggestion
    const cdLen = commands.length;
```